Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -908,7 +902,7 @@ + (void)startWithUserId:(NSString *)userId
     params[LP_PARAM_INBOX_MESSAGES] = [self.inbox messagesIds];
     
     // Push token.
-    NSString *pushTokenKey = [Leanplum pushTokenKey];
+    NSString *pushTokenKey = [[LPPushNotificationsManager sharedManager].handler pushTokenKey];
```

Comment:
could we move the persistence logic to the manager as well?

Reply:
all logic for push token is moved into the manager